Resources/Prototypes/Entities/Objects/Misc/identification_cards.yml

```diff
@@ -460,12 +460,84 @@
 - type: entity
   parent: CentcomIDCard
   id: ERTLeaderIDCard
-  name: ERT ID card
+  name: ERT leader ID card
   components:
   - type: Sprite
     layers:
     - state: gold
-    - state: ert_commander # we have the sprites but don't need individual ID entities for now.
+    - state: ert_commander
+  - type: IdCard
+    jobTitle: ERT Company Commander
+  - type: Item
+    heldPrefix: gold
+
+- type: entity
+  parent: CentcomIDCard
+  id: ERTChaplainIDCard
+  name: ERT Chaplain ID card
+  components:
+  - type: Sprite
+    layers:
+    - state: gold
+    - state: ert_chaplain # we have the sprite for the id but dont have chaplain ERT equipment for now.
+  - type: IdCard
+    jobTitle: ERT Soul Officer
+  - type: Item
+    heldPrefix: gold
```

Comment:
`  - type: Item`
`	 heldPrefix: gold`
    is inherited from the CentcomIDCard so these lines aren't needed iirc

Reply:
Unfortunately not.
![image](https://github.com/space-wizards/space-station-14/assets/107586145/b842a493-98ab-44ed-81c6-9888d7b1f8fd)
They were set on the Leader ID.
I suppose I could make a base item prototype, but that seems silly.

Reply:
hmm, maybe I can parent the leader ID anyway and just fill the jobTitle field?

Reply:
The leader is gold actually, so its sorta funky, but I parented off the chaplain ID.